api_tests/institutions/views/test_institution_relationship_nodes.py

```diff
@@ -1,303 +1,330 @@
+        assert res.status_code == 403
+        node_one.reload()
+        node_two.reload()
+        assert institution not in node_one.affiliated_institutions.all()
+        assert institution not in node_two.affiliated_institutions.all()
 
-    def test_add_some_existant_others_not(self):
-        assert_in(self.institution, self.node1.affiliated_institutions.all())
+    def test_add_some_existant_others_not(self, institution, node, app, url_institution_nodes, make_payload, user):
+        assert institution in node.affiliated_institutions.all()
 
-        node = NodeFactory(creator=self.user)
-        res = self.app.post_json_api(
-            self.institution_nodes_url,
-            self.create_payload(node._id, self.node1._id),
-            auth=self.user.auth
+        node_one= NodeFactory(creator=user)
+        res = app.post_json_api(
+            url_institution_nodes,
+            make_payload(node_one._id, node._id),
+            auth=user.auth
         )
 
-        assert_equal(res.status_code, 201)
+        assert res.status_code == 201
         node.reload()
-        self.node1.reload()
-        assert_in(self.institution, self.node1.affiliated_institutions.all())
-        assert_in(self.institution, node.affiliated_institutions.all())
+        node_one.reload()
+        assert institution in node.affiliated_institutions.all()
+        assert institution in node_one.affiliated_institutions.all()
 
-    def test_only_add_existent_with_mixed_permissions(self):
-        assert_in(self.institution, self.node1.affiliated_institutions.all())
-        assert_in(self.institution, self.node2.affiliated_institutions.all())
+    def test_only_add_existent_with_mixed_permissions(self, institution, node, node_public, app, url_institution_nodes, make_payload, user):
+        assert institution in node.affiliated_institutions.all()
+        assert institution in node_public.affiliated_institutions.all()
 
-        res = self.app.post_json_api(
-            self.institution_nodes_url,
-            self.create_payload(self.node2._id, self.node1._id),
+        res = app.post_json_api(
+            url_institution_nodes,
+            make_payload(node_public._id, node._id),
             expect_errors=True,
-            auth=self.user.auth
+            auth=user.auth
         )
 
-        assert_equal(res.status_code, 403)
-        self.node1.reload()
-        self.node2.reload()
-        assert_in(self.institution, self.node1.affiliated_institutions.all())
-        assert_in(self.institution, self.node2.affiliated_institutions.all())
+        assert res.status_code == 403
+        node.reload()
+        node_public.reload()
+        assert institution in node.affiliated_institutions.all()
+        assert institution in node_public.affiliated_institutions.all()
+
+    def test_only_add_existent_with_permissions(self, user, node, institution, app, url_institution_nodes, make_payload):
+        node_one = NodeFactory(creator=user)
```

Comment:
I'm mostly suggesting adding node_one (and possibly node_two if it is used more than once) as a second fixture since it is used in more than 1 test, not using the previously existing node fixture